cpan_deps_graph.pl

```diff
@@ -96,16 +104,22 @@
   }
 };
 
-helper get_dist_deps => sub ($c, $dist, $phases, $relationships, $perl_version) {
+helper get_dist_deps => sub ($c, $dist, $phases, $relationships, $perl_version, $dist_version = undef) {
   $perl_version = $perl_version->numify;
   my $redis = $c->redis->db;
   my %all_deps;
+  my $versioned_deps = $c->retrieve_dist_deps($dist, $dist_version)
```

Comment:
conditional 'my' declaration, and you can simplify the test to `length $dist_version` as this won't be running on perl 5.10

Reply:
I believe the new version fixes this? (My thinking was it's not a conditional declaration, but a conditional assignment, but it's up to you :-)

Reply:
any "my ... if ..." is problematic. LGTM